ios/JSProperties.swift

```diff
@@ -31,4 +33,10 @@ enum JSProperties: String {
         case programName
         case deleteInstructions
     }
+
+    enum CardConfig: String {
+        case id
+        case consentID
```

Comment:
Since I suggested [this](https://github.com/FidelLimited/rn-sdk/pull/111/files#r1219247046), please make sure that the value of this case will be consistent with the one received from JS ("consentId", not "consentID). Can be done like this:
`case consentID = "consentId"`